src/commcare_cloud/ansible/roles/formplayer/templates/monit.formplayer.j2

```diff
@@ -0,0 +1,5 @@
+ check file formplayer with path {{ formplayer_logging_dir }}/formplayer-spring.log
+  group formplayer
+  start program = "supervisorctl start {{ project }}-{{ deploy_env }}-formsplayer-spring"
```

Comment:
Made an assumption that the formplayer role has access to the `project` and `deploy_env` variables

Reply:
That assumption seems good, but you'll also be able to tell for sure when you run this (even in check mode)